sonic-net/sonic-buildimage
# Make macsec container to build with bookworm

#### Why I did it
We Nokia took the task to make macsec container to work with bookworm. 

This PR requires another PR for sonic-wpa-supplicant(https://github.com/sonic-net/sonic-wpa-supplicant/pull/80) to be in first. Otherwise sonic build will fail for bookworm wpasupplicant debian package.

#### How I did it


#### How to verify it
After build and boot up system with new image, macsec container should run with bookworm. Corresponding mgmt macsec suite will work here. 
#### Which release branch to backport (provide reason below if selected)

#### Tested branch (Please provide the tested image version)
Tested with latest master of sonic-buildimage. Built vs image and loaded into vs testing environment. Code pass with macsec test suites.

#### Description for the changelog
Make macsec container to build with bookworm
